docs/.well-known/security.txt

```diff
@@ -0,0 +1,43 @@
+-----BEGIN PGP SIGNED MESSAGE-----
+Hash: SHA512
+
+# Domeinen van de Rijksoverheid kunnen met een 302 redirect verwijzen naar
+# het centrale bestand op https://www.ncsc.nl/.well-known/security.txt
+# omdat het NCSC het centrale meldpunt is voor kwetsbaarheden en incidenten
+# voor de Rijksoverheid.
+#
+# Dutch central government domains can redirect to the central file located
+# at https://www.ncsc.nl/.well-known/security.txt with a 302 redirect,
+# because NCSC-NL is the central point of contact for vulnerabilities and
+# incidents for the Dutch central government.
+
+Expires: 2025-01-31T22:59:00.000Z
+Canonical: https://www.ncsc.nl/.well-known/security.txt
+
+Policy: https://www.ncsc.nl/contact/kwetsbaarheid-melden
```

Comment:
Moeten we security.md hier ook naar laten verwijzen ?

Reply:
I'm not sure. Is it ok that we make this another topic so we can continue with this PR?